plugins/repository-hdfs/src/main/java/org/elasticsearch/repositories/hdfs/HdfsSecurityContext.java

```diff
@@ -50,7 +50,7 @@
     static {
         // We can do FS ops with only a few elevated permissions:
         SIMPLE_AUTH_PERMISSIONS = new Permission[]{
-            new SocketPermission("*", "connect"),
+            new SocketPermission("*", "connect,resolve"),
```

Comment:
Why do we need `resolve` now, it seems the only new thing we do is a `.seek` call?

Reply:
likely not. I added this to try to appease the permission errors. Given the problem was something else, it's unlikely that we'll need this.